tests/e2e/pageobjects/ide/GitPlugin.ts

```diff
@@ -75,12 +83,19 @@ export class GitPlugin {
         await this.driverHelper.waitPresence(By.xpath(`//div[text()='Staged Changes']/parent::div/parent::div/parent::div/following-sibling::div//span[text()=\'${expectedStagedItem}\']`), timeout);
     }
 
-    async commitFromScmView(timeout: number = TimeoutConstants.TS_GIT_CONAINER_INTERACTION_TIMEOUT) {
+    async commitFromCommandMenu(timeout: number = TimeoutConstants.TS_GIT_CONAINER_INTERACTION_TIMEOUT) {
```

Comment:
The "timeout" argument is redundant here now. Please put it to the below methods or remove it.

Reply:
The timeout has been removed